search-service/src/test/kotlin/com/egm/stellio/search/web/TemporalEntityOperationsHandlerTests.kt

```diff
@@ -0,0 +1,216 @@
+package com.egm.stellio.search.web
+
+import com.egm.stellio.search.config.WebSecurityTestConfig
+import com.egm.stellio.search.model.TemporalQuery
+import com.egm.stellio.search.util.QueryUtils
+import com.egm.stellio.shared.util.*
+import com.egm.stellio.shared.util.JsonUtils.deserializeObject
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.*
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest
+import org.springframework.context.annotation.Import
+import org.springframework.security.test.context.support.WithAnonymousUser
+import org.springframework.security.test.context.support.WithMockUser
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.web.reactive.server.WebTestClient
+import org.springframework.web.reactive.function.BodyInserters
+import java.time.ZonedDateTime
+
+@ActiveProfiles("test")
+@WebFluxTest(TemporalEntityOperationsHandler::class)
+@Import(WebSecurityTestConfig::class)
+@WithMockUser
+class TemporalEntityOperationsHandlerTests {
```

Comment:
a lot of the tests here are globally the same than in `TemporalEntityHandlerTests`. if the code is better structured, you could here just focus on testing the specifics of this endpoint.